website/docs/documents-loading.mdx

```diff
@@ -11,18 +11,13 @@ Any input provided as a source will be recognized by utils automatically.
 
 It also extracts usages of `gql` from code files using [`@graphql-tools/graphql-tag-pluck`](/docs/graphql-tag-pluck).
 
-For notes on typescript, refer to [loaders](/docs/loaders)
```

Comment:
Actually this refers to the list of the loaders here;
https://www.graphql-tools.com/docs/schema-loading#loaders

Reply:
added it!

Reply:
Thanks <3